src/Raven.Server/Documents/Handlers/QueriesHandler.cs

```diff
@@ -335,7 +329,9 @@ await using (var writer = new AsyncBlittableJsonTextWriter(queryContext.Document
 
         private async Task ServerSideQuery(QueryOperationContext queryContext, RequestTimeTracker tracker, HttpMethod method)
         {
-            var indexQuery = await GetIndexQuery(queryContext.Documents, method, tracker);
+            var indexQueryReader = new IndexQueryReader(GetStart(), GetPageSize(), HttpContext, RequestBodyStream(), Database.QueryMetadataCache, Database);
```

Comment:
IndexQueryReader probably will be removed when we will introduce appropriate Processor? cc @karmeli87

Reply:
Done in https://github.com/ravendb/ravendb/pull/14460